koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -162,6 +162,19 @@ CREATE TABLE IF NOT EXISTS hive.{{schema | sqlsafe}}.reporting_ocpawscostlineite
 ) WITH(format = 'PARQUET', partitioned_by=ARRAY['aws_source', 'ocp_source', 'year', 'month', 'day'])
 ;
 
+{% if unattributed_storage %}
```

Comment:
Do we need to disable table creation? We didn't for Azure, I don't think.

Reply:
I put it behind unleash for now so that there is flexibility to add another column if needed later on in the process without having to run a trino migration. If it is a blocker I can remove the if though.